### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Bogdan-Stratiychuk**

### CODE REVIEW

File: src/ifElse.test.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('condition = true -> run first callback', () => {
+    const condition = ()=> true;
+
+    const first = jest.fn();
+    const second = jest.fn();
```
Comment (4ksandrey) [inline src/ifElse.test.js:11]: set variables names more informative, for example firstCallback and secondCallback
fix it everywhere please

File: src/ifElse.test.js [hunk 2]
```diff
@@ -1,11 +1,30 @@
 'use strict';
 
+const { ifElse } = require("./ifElse");
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('condition = true -> run first callback', () => {
```
Comment (4ksandrey) [inline src/ifElse.test.js:7]: test description should be understandable, try to rephrase this summary and for the second test

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
Check the comments

File: src/ifElse.test.js (hunk 1 above)
Comment (Bogdan-Stratiychuk) [reply src/ifElse.test.js]: fixed

File: src/ifElse.test.js (hunk 2 above)
Comment (Bogdan-Stratiychuk) [reply src/ifElse.test.js]: fixed

**Pull request closed without merging**